<ui version="4.0" >
 <class>FindPeopleFrameClass</class>
 <widget class="QFrame" name="FindPeopleFrameClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>461</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="frameShape" >
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow" >
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth" >
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="leftMargin" >
    <number>30</number>
   </property>
   <property name="rightMargin" >
    <number>30</number>
   </property>
   <property name="bottomMargin" >
    <number>30</number>
   </property>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>TopText</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>429</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <property name="verticalSpacing" >
      <number>1</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="searchNameLabel" >
       <property name="text" >
        <string>NameLoginOrEmail</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="searchNameEdit" />
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="findPeopleButton" >
       <property name="toolTip" >
        <string>FindButtonTip</string>
       </property>
       <property name="accessibleName" >
        <string>rectButton</string>
       </property>
       <property name="text" >
        <string>FindButton</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QCheckBox" name="advancedSearchCheckBox" >
       <property name="text" >
        <string>AdvancedSearchCheckBox</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="advancedSearchGroup" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <layout class="QGridLayout" name="advancedSearchGLayout" >
        <property name="margin" >
         <number>5</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="countryLabel" >
          <property name="text" >
           <string>Country</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="countryCombo" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLabel" name="genderLabel" >
          <property name="text" >
           <string>Gender</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <widget class="QComboBox" name="genderCombo" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>89</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>89</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="cityLabel" >
          <property name="text" >
           <string>City</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="cityEdit" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QLabel" name="ageRangeLlabel" >
          <property name="text" >
           <string>Age</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" >
         <widget class="QComboBox" name="ageRangeCombo" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>89</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>89</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5" >
         <spacer name="horizontalSpacer_5" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="foundUsersLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>FoundUsersCount_DT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameMid" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MoodBox::ContactTable" name="contactTable" >
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode" >
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
     <property name="gridStyle" >
      <enum>Qt::NoPen</enum>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled" >
      <bool>false</bool>
     </property>
     <property name="columnCount" >
      <number>4</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>MoodBox::ContactTable</class>
   <extends>QTableWidget</extends>
   <header>uitools.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchNameEdit</tabstop>
  <tabstop>findPeopleButton</tabstop>
  <tabstop>advancedSearchCheckBox</tabstop>
  <tabstop>countryCombo</tabstop>
  <tabstop>cityEdit</tabstop>
  <tabstop>genderCombo</tabstop>
  <tabstop>ageRangeCombo</tabstop>
  <tabstop>contactTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
